metrics-proxy/src/main/java/ai/vespa/metricsproxy/telegraf/TelegrafRegistry.java

```diff
@@ -12,11 +15,15 @@
 
     private static final List<Telegraf> telegrafInstances = Collections.synchronizedList(new ArrayList<>());
 
+    private static final Logger logger = Logger.getLogger(TelegrafRegistry.class.getName());
+
     public void addInstance(Telegraf telegraf) {
+        logger.log(LogLevel.DEBUG, () -> "Adding Telegraf instance to registry");
```

Comment:
Please add the instance to the log message. Nice to see the same hash go in and out.

Reply:
```suggestion
        logger.log(LogLevel.DEBUG, () -> "Adding Telegraf instance to registry: " + instance);
```